tests/e2e/scenarios/harvest_nucl_th_and_jlab_curation/LegacyService/allocaterecord.yaml

```diff
@@ -0,0 +1,19 @@
+request:
```

Comment:
same here, we can reuse

Reply:
IMO, is a bit bad having shared "fixtures" between tests...

Reply:
it's copy-pasted, not shared, that makes the git repo smaller, and speeds up each and every process that forks/clones it (any ci step, docker building, etc.). This one is small, but the pdf and such are quite noticeable.

Reply:
sure, got it :+1: